lib/pages/login-page.js

```diff
@@ -98,4 +98,11 @@ export default class LoginPage extends AsyncBaseContainer {
 	static getLoginURL() {
 		return dataHelper.getCalypsoURL( 'log-in' );
 	}
+
+	async setGdprCookie() {
+		const setCookieCode = function() {
```

Comment:
This doesn't seem to work when running against any other environment except wordpress.com (eg. calypso.localhost)